frontend/src/pages/article/article.tsx

```diff
@@ -27,16 +46,16 @@ const Article: FC = () => {
     { id: 5, name: 'Tech' },
   ];
 
-  const MOCKED_ARTICLE = {
-    title: 'Modern Full-Stack Developer Tech Stack 2021',
-    text: MOCKED_TEXT,
+  const ARTICLE = {
+    title,
+    text,
     tags: MOCKED_TAGS,
   };
 
   return (
     <Layout>
       <div className={styles.container}>
-        <ArticleView article={MOCKED_ARTICLE} />
+        <ArticleView article={ARTICLE} />
         {id && <Author />}
```

Comment:
You should join related author user data to fetched article, and use it here
And rename component please
```suggestion
        <AuthorDetails author={author} />
```

Reply:
Currently, I don't have author details, so I can't edit it. @PablloZz is working on it, and it will be possible to do it later.